### PR TITLE
Feature/34 simplify api

### PR DESCRIPTION
**Pull request by eddiejaoude**
### Simplified Code & API
- Removed relationships 
- Using Single Entity
- Updated Docs

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -11,39 +11,33 @@ API for the Member Service
    * `/filter` - Runs on every **Request** and injects Application Version (from `/resources/application.yml` Config) into Response Header
    * `/repository` - Manage the Domain Model
 * `src/main/resources`
-   * `aaplication.yml` - Application config
+   * `aaplication.yml` - Application config (contains `dev` & `prod`
```
Comment (brettminnie) [inline README.md]: Aapplication? Typo?

**Comment by brettminnie** [comment]
A lot cleaner :+1: only query is slepping :wink:

File: README.md (hunk 1 above)
Comment (eddiejaoude) [inline README.md]: Doh. Thanks :smile:

**Comment by eddiejaoude** [comment]
>  only query is slepping

What do you mean?

**Comment by brettminnie** [comment]
Was an intentional typo on spelling :)

**Comment by brettminnie** [comment]
Merged

**Comment by eddiejaoude** [comment]
> Was an intentional typo on spelling :)

Dont confuse me :p

**Comment by brettminnie** [comment]
Not even trying to :wink:

**Pull request merged**